framework/include/tinyalsa/tinyalsa.h

```diff
@@ -389,7 +409,7 @@ unsigned int pcm_bytes_to_frames(const struct pcm *pcm, unsigned int bytes);
  * @brief Gets the subdevice on which the pcm has been opened.
  *
  * @details @b #include <tinyalsa/tinyalsa.h>
- * @param[in] A PCM handle.
+ * @param[in] pcm A PCM handle.
```

Comment:
remove pcm please.
drop & drain too.

Reply:
Sorry, I dont understand.
I added pcm there because the previous comment said that @param must have the parameter name followed by the description.

Reply:
I didn't know that it has changed. nvm my comment.